lib/apiservers/portlayer/swagger.json

```diff
@@ -1748,6 +1748,63 @@
 				}
 			}
 		},
+		"/containers/{handle}/name": {
+			"put": {
```

Comment:
The google API uses PATCH quite liberally (https://cloud.google.com/compute/docs/reference/latest/). I think PATCH is the right thing to do here.